econml/inference/_inference.py

```diff
@@ -486,9 +498,13 @@ def intercept__inference(self, T):
             warn("Final model doesn't have a `intercept_stderr_` attribute. "
                  "Only point estimates will be available.")
             intercept_stderr = None
+        # d_t=1 here since we measure the effect across all Ts
+        # A dummy treatment names is passed in
```

Comment:
Consider rewording, since it is not actually passed in

```suggestion
        # Omit treatment name argument to fall back to the default
```